unit_test/entities/test_esdl_object.py

```diff
@@ -81,13 +80,13 @@ def test_get_connected_assets(self):
         """Test for connection of two assets."""
         # Arrange
         asset = self.esdl_object.get_all_assets_of_type("producer")[0].esdl_asset
-        test_list1 = [("Pipe1_ret", Port.Out)]
-        test_list2 = [("Pipe1", Port.In)]
+        test_list1 = [("Pipe1_ret", "935fb733-9f76-4a8d-8899-1ad8689a4b12")]
```

Comment:
Hard coded value can be asset.port[1].id

Reply:
aangepast